### ISSUE TITLE
Improve search calls

### ISSUE THREAD

**Issue by reddevilcero** [nabkd6/GiphyClonePairProgramming#6]
Need of improvements in calls to the api as more information for each gif will be needed (ex. title-slug source, user. etc.. ) and then create a view for each gif with its own information.

Do you thing is good idea or aproach.

**Comment by nabkd6** [comment]
Yeah probably. We need to decide how far we want to take it as well. Will have some limitations eventually because we don't want to actually create a database or store any real information--just rely on API calls I think. Alternative to this would be to just pop the GIF out of on the search page--and then have a button to copy link on the GIF.

**Comment by reddevilcero** [comment]
well that's true ,I thing just creating a function which store the last retrieve call count and make a new call with offset param updated we can go as far the user click next button or is scrolling, and we are working over a JSON file and a list or dict of plain text.

make your approach and we can discuss over it later.
PD: I won't we able to assist next MeetUp and no idea when it will possible to assist the next one.
